### PR TITLE
fixing up music.md for best practices on changing midi channel synths

### PR DESCRIPTION
**Pull request by bwhitman**

### CODE REVIEW

**Comment by bwhitman** [comment]
OK, this should be that. I tested `voices` changing patches without `music_map`, using `add_synth` instead, seems to work.

File: docs/music.md [hunk 1]
```diff
@@ -269,10 +287,17 @@ Now quit the `jam2` app if it was already running and re-`run` it. You should se
...
 
 ```python
+s = midi.OscSynth(wave=amy.SINE)
+s.note_on(60,1)
+s.note_off(60)
+```
+
+Let's try it as a sampler. There are 29 samples of drum-like and some instrument sounds in Tulip, and it can adjust the pitch and pan and loop of each one. You can try it out by just
 
+```python
```
Comment (dpwe) [inline docs/music.md:300]: In the next line: "after you specify how many voices you want" appears unnecessary

File: docs/tulip_api.md [hunk 2]
```diff
@@ -491,11 +491,11 @@ By default, Tulip boots into a live MIDI synthesizer mode. Any note-ons, note-of
 
 By default, MIDI notes on channel 1 will map to Juno-6 patch 0. And MIDI notes on channel 10 will play the PCM samples (like a drum machine).
 
-You can adjust which voices are sent with `tulip.music_map(channel, patch_number, voice_count)`. For example, you can have Tulip play DX7 patch 129 on channel 2 with `tulip.music_map(2, 129)`. The channel is a MIDI channel (we use 1-16 indexing), the patch_number is an AMY patch number, and voice_count is the optional number of voices (polyphony) you want to support for that channel and patch. 
+You can adjust which voices are sent with `midi.config.add_synth(channel, patch, polyphony)`. For example, you can have Tulip play DX7 patch 129 on channel 2 with `midi.config.add_synth(channel=2, patch=129, polyphony=1)`. The `2`, channel, is a MIDI channel (we use 1-16 indexing), the patch `129` is an AMY patch number, `1` is the optional number of voices (polyphony) you want to support for that channel and patch.
```
Comment (dpwe) [inline docs/tulip_api.md:494]: I don't think polyphony is optional.  We should probably rename it num_voices for consistency within the API, but I understand no code changes in this PR.

**Review by dpwe — APPROVED** [review]
LGTM

**Comment by bwhitman** [comment]
thank you !!!!!

**Pull request merged**